spacy_llm/tasks/templates/summarization.jinja

```diff
@@ -0,0 +1,29 @@
+You are an expert summarization system. Your task is to accept Text as input and summarize the Text in a concise way.
+{%- if max_n_words -%}
+{# whitespace #}
+The summary must not contain more than {{ max_n_words }} words.
+{%- endif -%}
+{# whitespace #}
+{%- if examples -%}
+Below are some examples (only use these as a guide):
+{# whitespace #}
+{%- for example in examples -%}
+{# whitespace #}
+Text:
+'''
+{{ example.text }}
+'''
+Summary:
+'''
+{{ example.text }}
```

Comment:
```suggestion
{{ example.summary }}
```

Reply:
If this correction is correct, could we have a unit test that would have captured this?

Reply:
Hm, we can check whether examples' summary is shorter than the text?

Reply:
That'd be a reasonable check, yep :-)

Reply:
I'm integrating the check into the task, to emit a warning if
- examples' summary is longer than the text or
- examples' summary is longer than `_max_n_words`.

For that I need to tokenize and count the words though, so I'd need to instantiate a pipeline in the task? :grimacing: 
The cheap alternative is to go by character count.

What's your take?

Reply:
Hm, `initialize()` is always called, so we have access to the original pipeline. We could run this with everything except the tokenizer disabled (and raise a warning if the pipeline doesn't have a tokenizer).

Reply:
GPT-3.5 seems to be considerably worse at counting chars than words, which makes sense, I guess.

Reply:
The cheap workaround I'll go with for now is to just split by spaces :upside_down_face:

Reply:
Added a check in https://github.com/explosion/spacy-llm/pull/181/commits/7a9c8af5765e3050dbeec49b79ca9736c540fb6c.